### DISCUSSION TITLE
Silly Question - Help!

### DISCUSSION THREAD

**Discussion by bassoonwebgirl** [community/community#49355] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Once I've created a repository and uploaded my files and made my turned my settings to 'public', how can I access and share my website?? Thanks

**Comment by rauzerfabiano** [comment]
If you have created a repository and uploaded your files to it, and set the repository as "public", you can access your website by visiting the URL https://[username].github.io/[repository-name] in a web browser, replacing [username] with your GitHub username and [repository-name] with the name of your repository.

For example, if your GitHub username is "johndoe" and your repository is named "my-website", you can access your website at https://johndoe.github.io/my-website.

To share your website with others, you can simply give them the URL. They should be able to access your website if it's set to "public".

**Comment by radar-blip** [comment]
This should help you: https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site#creating-your-site

It might be easier for you to bring your `index.html` out of the CSS folder (into root).
